pkg/frontend/assets/flowhouse.js

```diff
@@ -142,27 +142,58 @@ function drawChart() {
 }
 
 function renderChart(rdata) {
-  pres = Papa.parse(rdata.trim())
+  pres = Papa.parse(rdata.trim());
+
+  var filtered = [pres.data[0]];
+  for (var i = 1; i < pres.data.length; i++) {
+    var row = pres.data[i];
```

Comment:
[sand] use modern JS

```suggestion
  for (const row of pres.data) {
```

Reply:
ty, added